diff -urN simulavr-0.1.2.2.orig/doc/Makefile.am simulavr-0.1.2.2/doc/Makefile.am
--- simulavr-0.1.2.2.orig/doc/Makefile.am	2003-09-27 00:07:34.000000000 +0200
+++ simulavr-0.1.2.2/doc/Makefile.am	2007-08-04 14:15:56.000000000 +0200
@@ -48,7 +48,7 @@
 html: html/simulavr.html $(TARGET_DOX_HTML)
 
 html/simulavr.html: $(srcdir)/$(info_TEXINFOS)
-	texi2html -split_node $(srcdir)/$(info_TEXINFOS)
+	texi2html --output . -split_node $(srcdir)/$(info_TEXINFOS)
	if [ -e ./simulavr.html -o -e ./simulavr_1.html ]; then \
	  mkdir -p html ; \
	  mv *.html html ; \
	else \
